package-lock.json

```diff
@@ -0,0 +1,5187 @@
+{
```

Comment:
Well, yarn was a better solution than npm 3. And npm 5 was not yet released when we started this project